tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -12,9 +12,16 @@ module.exports = {
      * @param {string} fileName
      * @returns {string}
      */
-    getActionSelectorLowRes: function (action, fileName) {
-      return '(' + this.getFileRowSelectorByFileName(fileName) +
-        this.elements[action + 'ButtonInFileRow'].selector + ')[last()]'
+    getActionSelectorLowRes: function (itemName, action) {
```

Comment:
any specific reason to change the parameters around?

Reply:
I originally removed the fileName parameter at all and then just threw it back on the first position - will switch it back.